### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
@peternewman care to review this PR ?

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -14979,6 +14986,7 @@ unihabited->uninhabited
 unilateraly->unilaterally
 unilatreal->unilateral
 unilatreally->unilaterally
+uninitalize->uninitialize
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add the s versions of all of these.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Done

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Sorry @luzpaz , I wasn't very clear, I meant the British versions, i.e. uninitialise and friends.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: LOL, a-ok, I think I got it this time :rofl: f652e58

**Comment by luzpaz** [comment]
@larsoner should be good to go.

**Pull request merged**